CONTRIBUTING.md

```diff
@@ -417,17 +419,17 @@ wherever code is called that uses that citation (for example, in functions or in
 
 ### Installation
 
-Installation of PyBaMM and its dependencies is handled via [pip](https://pip.pypa.io/en/stable/) and [setuptools](http://setuptools.readthedocs.io/). It uses `CMake` to compile C++ extensions using [`pybind11`](https://pybind11.readthedocs.io/en/stable/) and [`casadi`](https://web.casadi.org/). The installation process is described in detail in the [source installation](https://docs.pybamm.org/en/latest/source/user_guide/installation/install-from-source.html) page and is configured through the `CMakeLists.txt` file.
+Installation of PyBaMM and its dependencies is handled via [pip](https://pip.pypa.io/en/stable/) and [scikit-build-core](https://scikit-build-core.readthedocs.io/en/latest/getting_started.html). It uses `CMake` to compile C++ extensions using [`pybind11`](https://pybind11.readthedocs.io/en/stable/) and [`casadi`](https://web.casadi.org/). The installation process is described in detail in the [source installation](https://docs.pybamm.org/en/latest/source/user_guide/installation/install-from-source.html) page and is configured through the `CMakeLists.txt` file.
 
 Configuration files:
 
 ```
-setup.py
 pyproject.toml
-MANIFEST.in
+.gitignore
```

Comment:
```suggestion
```

We can remove this, since `scikit-build-core` is configured entirely inside `pyproject.toml` and the reliance on a VCS for file inclusion/exclusion is purely a matter of preference. You've mentioned the use of `.gitignore` below, which seems to be sufficient.